DNN Platform/Modules/DnnExportImport/Components/Interfaces/IEntitiesController.cs

```diff
@@ -54,7 +54,7 @@ public interface IEntitiesController
         IList<ExportModulePermission> GetModulePermissions(int moduleId, DateTime toDate, DateTime? fromDate);
 
         IList<ExportTabModule> GetTabModules(int tabId, bool includeDeleted, DateTime toDate, DateTime? fromDate);
-        IList<ExportTabModuleSetting> GetTabModuleSettings(int tabId, DateTime toDate, DateTime? fromDate);
+        IList<ExportTabModuleSetting> GetTabModuleSettings(int tabId, bool includeDeleted, DateTime toDate, DateTime? fromDate);
```

Comment:
This is a breaking change. We need to keep the old signature and add a new method to the interface